src/RMGManager.cc

```diff
@@ -159,9 +162,21 @@ void RMGManager::Run() {
   }
 
   // eventually execute macros
-  for (const auto& macro : fMacroFileNames) {
-    RMGLog::Out(RMGLog::summary, "Loading macro file: ", macro);
-    UI->ApplyCommand("/control/execute " + macro);
+  for (const auto& macro : fMacroFilesOrContents) {
+    if (std::filesystem::exists(macro)) {
+      RMGLog::Out(RMGLog::summary, "Loading macro file: ", macro);
+      UI->ApplyCommand("/control/execute " + macro);
+    } else {
+      std::string expanded = macro;
+      std::replace(expanded.begin(), expanded.end(), ';', '\n');
+      std::replace(expanded.begin(), expanded.end(), ',', '\n');
```

Comment:
but why? Multiple macro commands could just be different args, no?

I am also not sure that nothing uses commas/semicola in macro command args in G4 which could horribly break...